hasher-matcher-actioner/setup.py

```diff
@@ -30,5 +30,8 @@
         "requests>=2.25.1",
     ],
     extras_require=extras_require,
-    entry_points={"console_scripts": ["hmacli = hmalib.scripts.cli.main:main"]},
+    entry_points={
+        "console_scripts": ["hmacli = hmalib.scripts.cli.main:main"],
+        "hmalib_extensions": ["echo = hmalib_extensions.main:print_arg"],
```

Comment:
Why is this step necessary? Why not make everything importable via their full path. Eg. instead of doing load_extension("echo"), I could be doing load_extension("hmalib_extensions.main:print_arg") instead.

Reply:
IMO the mappings is helpful in the context of creating the action on the UI or otherwise. It keeps things a little clean in if in the settings UI if you can specify the short and specific with `entry_point_name="echo"` for example

Reply:
> IMO the mappings is helpful in the context of creating the action on the UI or otherwise. It keeps things a little clean in if in the settings UI if you can specify the short and specific with `entry_point_name="echo"` for example

I get the brevity aspect, but in this case we had made it necessary for partners to make changes to a file that's checked in to version control. This will potentially make upgrading an issue. Urge you to think of an alternate way to retain the vanity -> actual method mapping that does not touch source control.